doc/flame/rendering/images.md

```diff
@@ -119,6 +119,45 @@ class MyGame extends Game {
 }
 ```
 
+## Loading images from the network
+
+Flame core package doesn't offer a built in method to loading images from the network.
+
+The reason for that is that Flutter/Dart does not have a built in http client, which required
+a package to be used and since there are a couple of packages available out there, we refrain
+from forcing, on even pushing the user in a specific package.
```

Comment:
Isn't `http` the "official" package? Its publisher is `dart.dev`, and it's described as
> Official core packages published by the Dart Team; these are foundational packages that complement the core libraries.

Reply:
It is, that is why that is the one that I choose for the default implementation on the package!

But it is not built in in flutter.